setup.py

```diff
@@ -26,7 +26,7 @@ def get_version():
         raise RuntimeError('Unable to find version string in {0}.'
                            .format(VERSION_FILE))
 
-install_requires = ['numpy', 'scanpointgenerator>=1.5', 'annotypes']
+install_requires = ['numpy', 'scanpointgenerator>=1.5', 'annotypes', 'vdsgen']
```

Comment:
Not install_requires. We need to define some optional requires targets for hdf5, pvaccess, etc. Cothread does need to be in this list now though...

Reply:
I've removed vdsgen and added cothread. Are the optional targets a future TODO? The imalcolm entry point does not list any of the extras_requires entries, so at present setup.py does not seem to bring in any extra dependencies.